src/main/java/com/asu/cse545/group12/controller/RegularUserController.java

```diff
@@ -295,5 +298,49 @@ public ModelAndView addSecurityQuestions(@ModelAttribute("form") Form form, Http
 		
 		
 	}
-
+	
+	@RequestMapping(value = "/viewExternalprofile")
+	public ModelAndView viewExternalprofile() {
+		if (logger.isDebugEnabled()) {
+			logger.debug("Adding security questions");
+		}
+		ModelAndView model = new ModelAndView();
+		
+		model.addObject("form", new Form());
+		model.setViewName("viewExternalprofile");
+		return model;
+	}
+	
+	@RequestMapping(value = "/viewExternalprofileform")
+	public ModelAndView getExternalprofile(@ModelAttribute("form") Form form, HttpServletRequest request) {
+		if (logger.isDebugEnabled()) {
+			logger.debug("Adding security questions");
+		}
+			ModelAndView model = new ModelAndView();
+			HttpSession session = request.getSession(false);
+			String requesterusername=(String) session.getAttribute("username");
+			Map samplemap = form.getMap();
+			String requestedtousername= (String) samplemap.get("userName");
+			
+			Users requestfromuser = userService.getUserByUserName(requesterusername);
+			int requesterUserId=requestfromuser.getUserId();
+			Users requesttouser = userService.getUserByUserName(requestedtousername);
+			int requesttoUserId=requesttouser.getUserId();
+			
+			List<Authorization> authorizationList = authorizationService.getAuthorizedNotifications(requesterUserId, requesttoUserId);
+			Authorization finalrequest = null;
+			Users user= new Users();
+			if(authorizationList!=null){
+				finalrequest = authorizationList.get(0);
+				 user = userService.getUserByUserName(requestedtousername);
+				finalrequest.setRequestStatus("INACTIVE");
```

Comment:
we are follow lowercase values as convention to make it sync everywhere.. so it should be inactive